src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector512.cs

```diff
@@ -1988,7 +2004,7 @@ public static Vector512<T> Min<T>(Vector512<T> left, Vector512<T> right)
         /// <returns>The product of <paramref name="left" /> and <paramref name="right" />.</returns>
         /// <exception cref="NotSupportedException">The type of <paramref name="left" /> and <paramref name="right" /> (<typeparamref name="T" />) is not supported.</exception>
         [Intrinsic]
-        public static Vector512<T> Multiply<T>(T left, Vector512<T> right) => left * right;
+        public static Vector512<T> Multiply<T>(T left, Vector512<T> right) => right * left;
```

Comment:
Cases like this aren't strictly needed, but it avoids needing +1 inlining step when we aren't accelerated and defers to the core algorithm instead.

This overload is just convenience so that `scalar * vector` and `vector * scalar` both work, since the operation is commutative, so there's no optimization opportunity missing here either.